src/components/Chef.js

```diff
@@ -0,0 +1,96 @@
+import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import { Link as LinkComponent } from 'react-router';
+import actions from '../action_creators';
+import constants from '../constants';
+
+const Link = createFactory(LinkComponent);
+const displayMax = 6;
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+function toOpenEntries(orderId, entry, entryIndex) {
+  return {
+    orderId,
+    entry,
+    entryIndex
+  };
+}
+
+class Chef extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.cancel = this.cancel.bind(this);
+    this.state = {
+      openEntries: [],
+      selectedEntry: undefined
+    };
+  }
+
+  setOpenEntries(orders) {
```

Comment:
I will eventually move this to the mapStateToProps to avoid doing this in both the componentWillMount and componentWillReceiveProps, but will need to do some refactoring to how we setup the store so we can use it in our test (based on what I've read). The refactor would allow us to stub the store in our test.